docs/toc.json

```diff
@@ -0,0 +1,346 @@
+    "type": "compute",
+    "nav": [{
+      "title": "Address",
+      "type": "compute/address"
+    }, {
+      "title": "Disk",
+      "type": "compute/disk"
+    }, {
+      "title": "Firewall",
+      "type": "compute/firewall"
+    }, {
+      "title": "Network",
+      "type": "compute/network"
+    }, {
+      "title": "Operation",
+      "type": "compute/operation"
+    }, {
+      "title": "Region",
+      "type": "compute/region"
+    }, {
+      "title": "Snapshot",
+      "type": "compute/snapshot"
+    }, {
+      "title": "VM",
+      "type": "compute/vm"
+    }, {
+      "title": "Zone",
+      "type": "compute/zone"
+    }]
+  }, {
+    "title": "DNS",
+    "type": "dns",
+    "nav": [{
+      "title": "Zone",
+      "type": "dns/zone"
+    }, {
+      "title": "Record",
+      "type": "dns/record"
+    }, {
+      "title": "Change",
+      "type": "dns/change"
+    }]
+  }, {
+    "title": "Datastore",
+    "type": "datastore",
+    "nav": [{
+      "title": "Dataset",
+      "type": "datastore/dataset"
+    }, {
+      "title": "Query",
+      "type": "datastore/query"
+    }, {
+      "title": "Transaction",
+      "type": "datastore/transaction"
+    }]
+  }, {
+    "title": "Logging",
+    "type": "logging",
+    "nav": [{
+      "title": "Sink",
+      "type": "logging/sink"
+    }, {
+      "title": "Log",
+      "type": "logging/log"
+    }, {
+      "title": "Entry",
+      "type": "logging/entry"
+    }]
+  }, {
+    "title": "Prediction",
+    "type": "prediction",
+    "nav": [{
+      "title": "Model",
+      "type": "prediction/model"
+    }]
+  }, {
+    "title": "PubSub",
+    "type": "pubsub",
+    "nav": [{
+      "title": "Topic",
+      "type": "pubsub/topic"
+    }, {
+      "title": "Subscription",
+      "type": "pubsub/subscription"
+    }]
+  }, {
+    "title": "Resource",
+    "type": "resource",
+    "nav": [{
+      "title": "Project",
+      "type": "resource/project"
+    }]
+  }, {
+    "title": "Search",
+    "type": "search",
+    "nav": [{
+      "title": "Index",
+      "type": "search/index"
+    }, {
+      "title": "Document",
+      "type": "search/document"
+    }, {
+      "title": "Field",
+      "type": "search/field"
+    }]
+  }, {
+    "title": "Storage",
+    "type": "storage",
+    "nav": [{
+      "title": "Bucket",
+      "type": "storage/bucket"
+    }, {
+      "title": "File",
+      "type": "storage/file"
+    }, {
+      "title": "Channel",
+      "type": "storage/channel"
+    }]
+  }, {
+    "title": "Translate",
+    "type": "translate"
+  }],
+  "types": [{
+    "id": "gcloud",
+    "title": "Node.js",
+    "contents": "index.json"
+  }, {
+    "id": "bigquery",
+    "title": "BigQuery",
+    "contents": "bigquery/index.json"
+  }, {
+    "id": "bigquery/dataset",
+    "title": "BigQuery » Dataset",
+    "contents": "bigquery/dataset.json"
+  }, {
+    "id": "bigquery/table",
+    "title": "BigQuery » Table",
+    "contents": "bigquery/table.json"
```

Comment:
I really like this step. I think `types` is a great name for this data. I'm disappointed I didn't think of it. :)

I do have a concern with the data as-is. We have unique IDs for guides and types, but not for methods. I don't see how we are able to link to a method in the `data-custom-type` attribute if we don't list the methods in the version manifest.

Currently, each type is defined as an ID, title, and a URL to the contents.

```json
}, {
  "id": "bigquery/table",
  "title": "BigQuery » Table",
  "contents": "bigquery/table.json"
}, {
```

This structure doesn't list methods and the method IDs. I'd like to see them listed on the same JSON object, so the angular app knows which file to load to get the information. Here is an example of what that might look like, with each method having an ID and a name:

```json
}, {
  "id": "bigquery/table",
  "title": "BigQuery » Table",
  "methods": [{
    "id": "bigquery/table.copy",
    "name": "copy"
  },{
    "id": "bigquery/table.create",
    "name": "create"
  },{
    "id": "bigquery/table.createReadStream",
    "name": "createReadStream"
  }],
  "contents": "bigquery/table.json"
}, {
```

If you have reservations about specifying both an ID and a name, only the ID could be provided, but I think that would hurt future uses, like for search:

```json
}, {
  "id": "bigquery/table",
  "title": "BigQuery » Table",
  "methods": ["bigquery/table.copy", "bigquery/table.create", "bigquery/table.createReadStream"],
  "contents": "bigquery/table.json"
}, {
```